### ISSUE TITLE
Stale scheme not applied if primary or accent colors are not configured

### ISSUE THREAD

**Issue by hSaria** [squidfunk/mkdocs-material#1714]
__I checked that...__

- [X] ... the documentation does not mention anything about my problem
- [X] ... the problem doesn't occur with the default MkDocs template
- [X] ... the problem is not in any of my customizations (CSS, JS, template)
- [X] ... there are no open or closed issues that are related to my problem

## Description

The `stale` color scheme (https://github.com/squidfunk/mkdocs-material/pull/1639) is not being applied.

### Expected behavior

The `stale` color scheme is applied to page.

### Actual behavior

The `stale` color scheme is not respected. A workaround is to configure the primary and/or accent colors. For example:

```yaml
theme:
  name: material
  palette:
    scheme: slate
    primary: indigo
```

### Steps to reproduce the bug

1. With the configuration below, browse to any page
2. The `stale` theme is not applied
### Package versions

* Python: 3.7.7
* MkDocs: 1.1.2
* Material: 5.2.0

### Project configuration

``` yaml
site_name: Example
site_url: https://example.net/
site_author: Example
repo_url: https://github.com/example/example/

theme:
  name: material
  palette:
    scheme: slate

nav:
  - Example: 'index.md'
```

### System information

* OS: macOS Catalina 10.15.4
* Browser: Safari 13.1

**Comment by squidfunk** [comment]
Fixed in 4c74ac25

**Comment by squidfunk** [comment]
Released as part of 5.2.1.
